modules/civicrmtheme/civicrmtheme.install

```diff
@@ -33,28 +33,28 @@
  *
  */
 
-/*
- * Implemenation of hook_enable()
+/**
+ * Implements hook_enable().
  */
 function civicrmtheme_enable() {
   // update module weight to 110, to make it run after system and civicrm
   db_query("UPDATE {system} SET weight = 110 WHERE name = 'civicrmtheme'");
 }
 
 /**
- * Implementation of hook_install().
+ * Implements hook_install().
  */
 function civicrmtheme_install() {
-  $link = l(t('Administer > Appearance > Administration theme'), 'admin/appearance');
-  drupal_set_message(t("CiviCRM theme configuration setting is available under !link", array('!link' => $link)));
+  $t = get_t();
```

Comment:
More information on this can be found https://www.drupal.org/docs/7/api/localization-api/when-to-use-st-t-and-get_t-in-install-and-update-functions